src/messenger-shortcuts.js

```diff
@@ -129,7 +129,8 @@ function focusMessageInput() {
 }
 
 function selectFirstSearchResult() {
-  var first = document.querySelector('ul[role="listbox"] li div');
+  var contacts = document.querySelectorAll('ul[role="listbox"]')[1];
```

Comment:
Ooh that's a good catch, I'll add a condition based on the number of characters in the search box.